### PR TITLE
fix(friends): dispatch update store event on denied and accept request

### PR DESCRIPTION
**Pull request by AlexZakablukov**
… request

**What this PR does** 📖
dispatch update store event on denied and accept request

**Which issue(s) this PR fixes** 🔨
[AP-1409](https://satellite-im.atlassian.net/browse/AP-1409)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: libraries/Solana/FriendsProgram/FriendsProgram.ts [hunk 1]
```diff
@@ -403,6 +403,11 @@ export default class FriendsProgram extends EventEmitter {
         toPaddedBuffer,
       ),
     )
+    console.log('acceptFriendRequest')
```
Comment (Jekrimo) [inline libraries/Solana/FriendsProgram/FriendsProgram.ts:406]: Do we need all these consoles?

**Review by Jekrimo — CHANGES_REQUESTED** [review]

File: libraries/Solana/FriendsProgram/FriendsProgram.ts (hunk 1 above)
Comment (AlexZakablukov) [reply libraries/Solana/FriendsProgram/FriendsProgram.ts]: removed

**Review by phillsatellite — APPROVED** [review]

**Review by Jekrimo — APPROVED** [review]

**Comment by iltumio** [comment]
we should close this PR since we are integrating the new version of the friends program here https://github.com/Satellite-im/Core-PWA/pull/2469

**Comment by stavares843** [comment]
closing due to @iltumio comment above

**Pull request closed without merging**